tests/test_market.py

```diff
@@ -1,35 +1,37 @@
 """Testing for the ElfPy package market module methods"""
-from __future__ import annotations  # types are strings by default in 3.11
+from __future__ import annotations
 
 import unittest
-
-import numpy as np
+from collections import defaultdict
 
 import elfpy.pricing_models.hyperdrive as hyperdrive_pm
 import elfpy.pricing_models.yieldspace as yieldspace_pm
 import elfpy.markets.hyperdrive.hyperdrive_market as hyperdrive_market
 import elfpy.markets.borrow as borrow
 import elfpy.time as time
+from elfpy.math import FixedPoint
+
+# TODO: remove this after FixedPoint PRs are finished
 
 
-class BaseMarketTest(unittest.TestCase):
+class MarketTest(unittest.TestCase):
     """Generic Parameter Test class"""
 
-    # TODO: Switching to fixed point or 64 bit float should allow us to increase this to WEI
+    # TODO: This approx should be much lower
     # issue #112
-    APPROX_EQ: float = 1e-15
+    APPROX_EQ: FixedPoint = FixedPoint(1e-6)
```

Comment:
is this strange/concerning that our precision worsened? maybe because previously we were using Decimal with very high (>18) decimals of precision?

Reply:
maybe to be expected of internal checks.
a check vs. the solidity result would be totally different, hopefully to no discrepancy.